nestkernel/archiving_node.cpp

```diff
@@ -100,7 +109,8 @@ nest::Archiving_Node::get_K_value( double t )
   int i = history_.size() - 1;
   while ( i >= 0 )
   {
-    if ( t > history_[ i ].t_ )
+    if ( t - history_[ i ].t_ > 1.0
+        * kernel().connection_manager.get_stdp_eps() )
```

Comment:
Could you try to get the -1.0 on the same line as `kernel()....`? I think that would be good for legibility.

Reply:
It's a +1.0 and I have removed it.